worker/runner.gen.go

```diff
@@ -435,32 +435,32 @@ func WithRequestEditorFn(fn RequestEditorFn) ClientOption {
 
 // The interface specification for the client above.
 type ClientInterface interface {
-	// AudioToTextAudioToTextPostWithBody request with any body
-	AudioToTextAudioToTextPostWithBody(ctx context.Context, contentType string, body io.Reader, reqEditors ...RequestEditorFn) (*http.Response, error)
+	// GenAudioToTextWithBody request with any body
+	GenAudioToTextWithBody(ctx context.Context, contentType string, body io.Reader, reqEditors ...RequestEditorFn) (*http.Response, error)
 
 	// HealthHealthGet request
 	HealthHealthGet(ctx context.Context, reqEditors ...RequestEditorFn) (*http.Response, error)
```

Comment:
@victorges this is what the https://github.com/oapi-codegen/oapi-codegen package creates when the `operation_id` is not set. We did fix this in the past using the [use_route_names_as_operation_ids](https://github.com/livepeer/ai-worker/blob/36d796d83646c1bd16e6ee21cde7789e6f378d4d/runner/app/main.py#L105-L108) function. I will leave it to you to decide if we want to use their convention and make the operation_id. healthGet or keep it as Health. I think Health is cleaner.